packages/ovh-manager/translations/Messages_cs_CZ.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
import only french file, others will be retrieved when we will do the first "retrieve" script ;)